ui/page/search/view.jsx

```diff
@@ -40,14 +44,16 @@ export default function SearchPage(props: Props) {
       <section className="search">
         {urlQuery && (
           <Fragment>
-            {isValid && (
-              <header className="search__header">
-                <ClaimUri uri={uri} />
-                <div className="card">
-                  <ClaimPreview uri={uri} type="large" placeholder="publish" />
-                </div>
-              </header>
-            )}
+            <header className="search__header">
+              <ClaimUri uri={Boolean(isModifiedUriValid) && normalizedModifiedUri} />
```

Comment:
I assume this isn't rendered if it's false? might be slightly cleaner/obvious to do

`{ isModifiedUriValid && <ClaimUri uri={normalizedModifiedUri} /> }`